### PR TITLE
Control render using configuration

### PR DESCRIPTION
**Pull request by BeyondEvil**
Fixes: #239

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -240,9 +240,16 @@ Display options
...
 By default, all rows in the **Results** table will be expanded except those that have :code:`Passed`.
 
-This behavior can be customized with a query parameter: :code:`?collapsed=Passed,XFailed,Skipped`.
+This behavior can be customized either with a query parameter: :code:`?collapsed=Passed,XFailed,Skipped`
+or by setting the :code:`render_collapsed` in a configuration file (pytest.ini, setup.cfg, etc).
 
+.. code-block:: ini
 
+  [pytest]
+  render_collapsed = True
```
Comment (nicoddemus) [inline README.rst]: Is there an extra character after `True` here?

File: README.rst [hunk 2]
```diff
@@ -240,9 +240,16 @@ Display options
...

+
+**NOTE:** Setting :code:`render_collapsed` will, unlike the query parameter, affect all statuses.
+
```
Comment (nicoddemus) [inline README.rst]: Here too

File: README.rst (hunk 1 above)
Comment (BeyondEvil) [reply README.rst]: Damn, you're right. 🤔 

1. How did you see them?
2. Any idea where they come from? They weren't visible characters, just some weird ASCII(maybe?).

Comment (nicoddemus) [reply README.rst]: No idea :(

Comment (BeyondEvil) [reply README.rst]: How did you notice/see them? I can't tell in the Github diff.

Comment (nicoddemus) [reply README.rst]: Weird they don't show up anymore... 🤔

Comment (BeyondEvil) [reply README.rst]: I fixed them @nicoddemus :) (the second commit)

Comment (nicoddemus) [reply README.rst]: Great, that's fixed then

**Comment by Wramberg** [comment]
@BeyondEvil As agreed in #239 I gave this a shot. The render is must faster now but unfortunately the collapsed class is misplaced. This means it is no longer possible to unfold the details/extras. With this PR, setting render_collapsed to true yields
```html
<tr>
  <td class="extra collapsed" colspan="4">
    <div>
```
but it should be
```html
<tr class="collapsed">
  <td class="extra" colspan="4">
    <div>
```
By changing the implementation to the following it works for me
```python
            if len(cells) > 0:
                tr_class = None
                if self.config.getini("render_collapsed"):
                    tr_class = "collapsed"

                self.row_table = html.tr(cells)
                self.row_extra = html.tr(
                    html.td(self.additional_html, class_="extra", colspan=len(cells)), class_=tr_class
                )
```
Essentially just moving the collapsed class to the parent `html.tr()`.

**Comment by BeyondEvil** [comment]
> @BeyondEvil As agreed in #239 I gave this a shot. The render is must faster now but unfortunately the collapsed class is misplaced. This means it is no longer possible to unfold the details/extras. With this PR, setting `render_collapsed=true` yields
> 
> ```
> <tr>
>   <td class="extra collapsed" colspan="4">
>     <div>
> ```
> 
> but it should be
> 
> ```
> <tr class="collapsed">
>   <td class="extra" colspan="4">
>     <div>
> ```
> 
> By changing the implementation to the following it works for me
> 
> ```python
>             if len(cells) > 0:
>                 tr_class = None
>                 if self.config.getini("render_collapsed"):
>                     tr_class = "collapsed"
> 
>                 self.row_table = html.tr(cells)
>                 self.row_extra = html.tr(
>                     html.td(self.additional_html, class_="extra", colspan=len(cells)), class_=tr_class
>                 )
> ```
> 
> Essentially just moving the collapsed class to the parent `html.tr()`.

lol, awkward.

Good catch! In the end it was good, because I noticed JS needed an update as well to display the correct CSS.

**Comment by BeyondEvil** [comment]
**Released in v2.1.0**

**Pull request merged**